frontend/Dockerfile

```diff
@@ -1,15 +1,23 @@
-FROM jfloff/alpine-python:3.6
-
+FROM femtopixel/google-closure-compiler:0.1.5 as builder
 LABEL maintainer="Dimitrios Mavrommatis <jim.mavrommatis@gmail.com>"
 
+WORKDIR /root
+COPY ./webapp/static/js/custom/ /root/
+COPY ./minify-js.sh /root/
+RUN mkdir /root/prod/
+RUN ./minify-js.sh
+
+FROM jfloff/alpine-python:3.6
+
 WORKDIR /root
 
 COPY . ./
+RUN rm /root/webapp/static/js/custom/*
+COPY --from=builder /root/prod/ /root/webapp/static/js/custom/
 
 RUN apk update && apk add --no-cache openssl-dev libffi-dev py-openssl sqlite-dev sqlite
 RUN pip install --upgrade 'pip<19.0'
 RUN pip --no-cache-dir install -r requirements.txt
-RUN pip --no-cache-dir install gunicorn
```

Comment:
Why is gunicorn deleted from here? It is installed only as a Python package from now on?

Reply:
exactly the same.. this one was installing the latest version